crates/binjs_io/src/entropy/dictionary.rs

```diff
@@ -246,14 +245,17 @@ where
     /// If the value is already in the cache, return `Fetch::Hit(index)`, where `index` is the
     /// immutable index of the value. Otherwise, allocate a new slot `index` and return
     /// `Fetch::Miss(index)`.
-    pub fn fetch_index(&mut self, value: &T) -> Fetch {
+    ///
+    /// In both `Fetch::Hit(index)` and `Fetch::Miss(index)`, `0` is the first value, `1` the second
+    /// value, etc. Indices returned with `Fetch::Hit` and `Fetch::Miss` will tend to overlap.
```

Comment:
can you explain some more about "overlap" ?
(otherwise this comment is just confusing)